transparenciagovbr/spiders/pagamento/spiders.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+
+import rows
+
+from transparenciagovbr import settings
+from transparenciagovbr.spiders.base import TransparenciaBaseSpider
+from transparenciagovbr.utils.date import today
+
+
+class BrazilianDateField(rows.fields.DateField):
+    INPUT_FORMAT = "%d/%m/%Y"
+
+
+class MoneyRealField(rows.fields.DecimalField):
+    @classmethod
+    def deserialize(cls, value):
+        """
+        >>> MoneyRealField.deserialize("89188,11")
+        '89188.11'
+        """
+        value = value.replace(",", ".")
+        return super().deserialize(value)
+
+
+SCHEMA_PATH = str((settings.REPOSITORY_PATH / "schema" / "pagamento.csv").absolute())
+SCHEMA = rows.utils.load_schema(
+    SCHEMA_PATH,
+    context={
+        "date": BrazilianDateField,
+        "text": rows.fields.TextField,
+        "integer": rows.fields.IntegerField,
+        "money_real": MoneyRealField,
+    },
+)
+FIELD_MAPPING = {
+    row.original_name: row.field_name for row in rows.import_from_csv(SCHEMA_PATH)
+}
```

Comment:
I think we could move the code from `BrazilianDateField` to here to a shared module so we avoid repetition (may transform these variables into function which receive the filename as parameter and define the parameter inside the class). What do you think?

Reply:
Thank you for the input :) I really liked your ideas and implemented them. Let me know if you liked it!